PROTOCOL.md

```diff
@@ -564,22 +565,50 @@ The following is an example `rowIdHighWaterMark` action:
 }
 ```
 
+### Domain Metadata
+The domain metadata action contains a configuration (string key/value pairs) for a named metadata domain. Two overlapping transactions conflict if they both contain a domain metadata action for the same metadata domain.
+
+There are two types of metadata domains:
```

Comment:
There seems to be some mixing and matching of "domain metadata" vs. "metadata domains."

I _think_ "metadata domain" is the concept (the ability to track some domain-specific metadata), and "domain metadata" is the corresponding log cation (`domainMetadata`)? 

Or is it just mixed and we should harmonize the wording?

Reply:
As you mentioned, `metadata domain` is more like a concept and its metadata is stored into the action named `DomainMetadata`. I made this intentionally so that readers (at least to me) can easier understand `User-controlled metadata domain` and `system-controlled metadata domain`. The word `domain` is matched with a feature